lib/PuppeteerSharp/.editorconfig

```diff
@@ -0,0 +1,173 @@
+[*.cs]
```

Comment:
I recall having issues with `.editorconfig` in other environments. Does it work on CI/CD? Can we force an error to see if that works?

Reply:
`.editorconfig` works with CI, see now failing pipelines.

Maybe the bad experience you previously had was CI pipelines with older build tools, where `.editorconfig` wasn't yet supported?
E.g. Visual Studio has [supported `.editorconfig`](https://learn.microsoft.com/en-us/visualstudio/code-quality/analyzers-faq?view=vs-2022) since v16.3 released in [September 2019](https://learn.microsoft.com/en-us/visualstudio/releases/2019/release-notes-v16.3#16.3.0)